containers/unit_tests/TestDynViewAPI.hpp

```diff
@@ -130,7 +130,7 @@ struct TestViewOperator_LeftAndRight<DataType, DeviceType, 7> {
                 for (unsigned i0 = 0; i0 < unsigned(left.extent(0)); ++i0) {
                   const long j = &left(i0, i1, i2, i3, i4, i5, i6) -
                                  &left(0, 0, 0, 0, 0, 0, 0);
-                  if (j <= offset || left_alloc <= j) {
+                  if (j < offset || left_alloc < j) {
```

Comment:
The second part checks that the pointer distance is always less than the allocation size, so this should still use `<=`. The same in all the other cases.

Reply:
Thanks, marking as resolved.